python/src/DistFunc_doc.i.in

```diff
@@ -583,185 +675,1304 @@ Examples
+
+with :math:`\nu > 0`.
+
+Examples
+--------
+>>> import openturns as ot
+>>> cdf = ot.DistFunc.pStudent(2.5, 0.9)
+>>> cdf = ot.DistFunc.pStudent(2.5, 0.9, True)"
+
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::DistFunc::rStudent
+"Realization of a scalar Student distribution.
+
+Parameters
+----------
+nu : float, :math:`\nu > 0`
+    The :math:`\nu` parameter.
+size : int
+    The number of realizations (optional).
+    
+    By default, size is equal to 1.
+
+Returns
+-------
+realization : float or a sequence of float if size is greater than 1.
+
+Notes
+-----
+The probability density function is defined as:
+
+.. math::
+    p(x) = \dfrac{1}{\sqrt{\nu}B(1/2, \nu/2)} \left(1+ \dfrac{x^2}{\nu}\right)^{-\dfrac{\nu+1}{2}} \quad x \in \Rset
+
+with :math:`\nu > 0`.
+
+
+Examples
+--------
+>>> import openturns as ot
+>>> ot.RandomGenerator.SetSeed(0)
+>>> r = ot.DistFunc.rStudent(2.5)
+>>> r = ot.DistFunc.rStudent(2.5, 10)"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::DistFunc::eZ1
+"Expectation of the min of n independent standard normal random variables.
+
+Parameters
+----------
+n : int
+
+Returns
+-------
+mu : float
+    The mean value of the min of n independent standard normal random variables."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::DistFunc::rUniformSegment
+"Realization of a uniform distribution over a segment.
+
+Parameters
+----------
+a : sequence of float, in :math:`\Rset^n`
+    The first vertex of the segment.
+b : sequence of float, in :math:`\Rset^n`
+    The second vertex of the segment.
+size : int
+    The number of realizations to generate.
+
+Notes
+-----
+This method samples the barycentric coordinates, allowing to work in any dimension.
+
+Returns
+-------
+realizations : :class:`~openturns.Point` or :class:`~openturns.Sample`
+    The realizations of the uniform disctribution.
+
+Examples
+--------
+>>> import openturns as ot
+>>> ot.RandomGenerator.SetSeed(0)
+>>> a = [1, 5]
+>>> b = [2, 3]
+>>> r = ot.DistFunc.rUniformSegment(a, b)
+>>> r = ot.DistFunc.rUniformSegment(a, b, 10)"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::DistFunc::rUniformTriangle
+"Realization of a uniform distribution over a triangle.
+
+Parameters
+----------
+a : sequence of float, in :math:`\Rset^n`
+    The first vertex of the triangle.
+b : sequence of float, in :math:`\Rset^n`
+    The second vertex of the triangle.
+c : sequence of float, in :math:`\Rset^n`
+    The third vertex of the triangle.
+size : int
+    The number of realizations to generate.
+
+Notes
+-----
+This method samples the barycentric coordinates, allowing to work in any dimension.
+
+Returns
+-------
+realizations : :class:`~openturns.Point` or :class:`~openturns.Sample`
+    The realizations of the uniform disctribution.
```

Comment:
```suggestion
    The realizations of the uniform distribution.
```

Reply:
This typo is everywhere in this file, so you need to CTRL-F this.

Reply:
yes...